card/mod/core/chunk/view_stub.rb

```diff
@@ -48,6 +48,10 @@ def interpret_override
         def process_chunk
           @processed = yield @stub_hash
         end
+
+        def result
+          @processed
+        end
```

Comment:
not sure how this got here, but I remember that this patch was to make it so that views could return nil without confusing the view cache.